hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -535,9 +535,6 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 			return ctrl.Result{}, fmt.Errorf("failed to reconcile KubevirtMachineTemplate: %w", err)
 		}
 		span.AddEvent("reconciled kubevirtmachinetemplate", trace.WithAttributes(attribute.String("name", machineTemplate.GetName())))
-	case hyperv1.NonePlatform:
-		// TODO: When fleshing out platform None design revisit the right semantic to signal this as conditions in a NodePool.
```

Comment:
please move this change to the commit that introduced so we can drop this commit

Reply:
Done